Documentation/planning/arcade-powered-source-build/implementation-plan.md

```diff
@@ -0,0 +1,42 @@
+# Source-build 5.0 Implementation Plan
+
+To get each repo building with the new source-build 5.0 plan, [Arcade-Powered Source-Build](https://github.com/dotnet/source-build/tree/release/3.1/Documentation/planning/arcade-powered-source-build), the following stages will be accomplished.  Below, the repos are placed in a table representing the earliest dependencies are available and work can begin.  Stages can be completed in parallel, but stage 4 may depend on output from stage 6 from previous repos.  The following graph and table will be used to track progress for each repo.
+
+> ![](img/implementation-plan-graph.png)
+> [source (img/implementation-plan-graph.dgml)](img/implementation-plan-graph.dgml)
+
+| Tier | Repo | (Stage 1) Build from source - 5.0 | (Stage 2) Patch Removal | (Stage 3) Move build params to eng dir | (Stage 4) Input Intermediate Packages Available | (Stage 5) Official build | (Stage 6) Source-build PR validation | (Stage 7) Prebuilt check / enforcement |
```

Comment:
Simplify stage names for less distracting wrapping with GitHub's width limit?

```suggestion
| Tier | Repo | (1) Build from source - 5.0 | (2) Patch Removal | (3) Move build params to eng dir | (4) Input Intermediate Packages Available | (5) Official build | (6) Source-build PR validation | (7) Prebuilt check / enforcement |
```

Reply:
I just changed them to include "stage" to better match the legend on the graph.  I'm going to try multi-line headers to see if this helps.